e2e/tests/authflow/models.go

```diff
@@ -1,10 +1,21 @@
 package tests
 
 type TestCase struct {
-	Name    string       `yaml:"name"`
-	Project string       `yaml:"project"`
-	Steps   []Step       `yaml:"steps"`
-	Before  []BeforeHook `yaml:"before"`
+	Name               string             `yaml:"name"`
+	Path               string             `yaml:"path"`
+	Focus              bool               `yaml:"focus"`
```

Comment:
What is Focus?

Reply:
It's created like https://pkg.go.dev/github.com/smartystreets/goconvey/convey#FocusConvey focus on one test case

Reply:
Please leave a comment above Focus for future readers.